jest.config.js

```diff
@@ -0,0 +1,11 @@
+import { createJsWithTsEsmPreset } from "ts-jest";
+
+const tsJestTransformCfg = createJsWithTsEsmPreset();
+
+process.env.DATABASE_URL ??= "postgres://postgres:illnevertell@db:5432/nutrien_test_1";
```

Comment:
Would absolutely be handled better by a `.env.test` file but the Jest+ESM issues consumed the time that would've otherwise been allocated to getting that set up.